Modules/Core/ImageFunction/include/itkWindowedSincInterpolateImageFunction.h

```diff
@@ -46,7 +46,7 @@ class ITK_TEMPLATE_EXPORT CosineWindowFunction
 
 private:
   /** Equal to \f$ \frac{\pi}{2 m} \f$ */
-  static const double m_Factor;
+  static constexpr double m_Factor = itk::Math::pi / (2 * VRadius);
```

Comment:
Your proposed change looks OK to me, but as it appears that `m_Factor` is only used inside `operator()`, I think it should just be a local (`static constexpr`) variable of `operator()`.

Reply:
Done.